src/components/Calculator.js

```diff
@@ -1,36 +1,74 @@
+            <span>{result}</span>
           </div>
           <div className="symbols">
-            <button className="btn" type="button">AC</button>
-            <button className="btn" type="button">+/-</button>
-            <button className="btn" type="button">%</button>
-            <button className="btn orange" type="button">÷</button>
-            <button className="btn" type="button">7</button>
-            <button className="btn" type="button">8</button>
-            <button className="btn" type="button">9</button>
-            <button className="btn orange" type="button">x</button>
-            <button className="btn" type="button">4</button>
-            <button className="btn" type="button">5</button>
-            <button className="btn" type="button">6</button>
-            <button className="btn orange" type="button">-</button>
-            <button className="btn" type="button">1</button>
-            <button className="btn" type="button">2</button>
-            <button className="btn" type="button">3</button>
-            <button className="btn orange" type="button">+</button>
-            <button className="btn zero" type="button">0</button>
-            <button className="btn" type="button">.</button>
-            <button className="btn orange" type="button">=</button>
+            <button className="btn" type="button" onClick={this.btnClick}>AC</button>
+            <button className="btn" type="button" onClick={this.btnClick}>+/-</button>
+            <button className="btn" type="button" onClick={this.btnClick}>%</button>
+            <button className="btn orange" type="button" onClick={this.btnClick}>÷</button>
+            <button className="btn" type="button" onClick={this.btnClick}>7</button>
+            <button className="btn" type="button" onClick={this.btnClick}>8</button>
+            <button className="btn" type="button" onClick={this.btnClick}>9</button>
+            <button className="btn orange" type="button" onClick={this.btnClick}>x</button>
+            <button className="btn" type="button" onClick={this.btnClick}>4</button>
+            <button className="btn" type="button" onClick={this.btnClick}>5</button>
+            <button className="btn" type="button" onClick={this.btnClick}>6</button>
+            <button className="btn orange" type="button" onClick={this.btnClick}>-</button>
+            <button className="btn" type="button" onClick={this.btnClick}>1</button>
+            <button className="btn" type="button" onClick={this.btnClick}>2</button>
+            <button className="btn" type="button" onClick={this.btnClick}>3</button>
+            <button className="btn orange" type="button" onClick={this.btnClick}>+</button>
+            <button className="btn zero" type="button" onClick={this.btnClick}>0</button>
+            <button className="btn" type="button" onClick={this.btnClick}>.</button>
+            <button className="btn orange" type="button" onClick={this.btnClick}>=</button>
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻